code/modules/admin/admin_verbs.dm

```diff
@@ -110,8 +110,6 @@ var/list/admin_verbs_event = list(
 	/client/proc/toggle_ert_calling,
 	/client/proc/show_tip,
 	/client/proc/cmd_admin_change_custom_event,
-	/client/proc/cmd_admin_custom_event_info,
-	/client/proc/cmd_view_custom_event_info,

```

Comment:
Why did you remove this

Reply:
Because those procs don't exist anymore?

Reply:
Oh I can’t read